Podfile.lock

```diff
@@ -4,21 +4,21 @@ PODS:
     - Alamofire (~> 4.8)
   - AlamofireNetworkActivityIndicator (2.4.0):
     - Alamofire (~> 4.8)
-  - AppAuth (1.6.0):
-    - AppAuth/Core (= 1.6.0)
-    - AppAuth/ExternalUserAgent (= 1.6.0)
-  - AppAuth/Core (1.6.0)
-  - AppAuth/ExternalUserAgent (1.6.0):
+  - AppAuth (1.6.1):
+    - AppAuth/Core (= 1.6.1)
+    - AppAuth/ExternalUserAgent (= 1.6.1)
+  - AppAuth/Core (1.6.1)
+  - AppAuth/ExternalUserAgent (1.6.1):
     - AppAuth/Core
-  - AppCenter (4.4.1):
-    - AppCenter/Analytics (= 4.4.1)
-    - AppCenter/Crashes (= 4.4.1)
-  - AppCenter/Analytics (4.4.1):
+  - AppCenter (4.4.3):
+    - AppCenter/Analytics (= 4.4.3)
+    - AppCenter/Crashes (= 4.4.3)
+  - AppCenter/Analytics (4.4.3):
     - AppCenter/Core
-  - AppCenter/Core (4.4.1)
-  - AppCenter/Crashes (4.4.1):
+  - AppCenter/Core (4.4.3)
+  - AppCenter/Crashes (4.4.3):
     - AppCenter/Core
-  - AppCenter/Distribute (4.4.1):
+  - AppCenter/Distribute (4.4.3):
     - AppCenter/Core
   - Automattic-Tracks-iOS (0.13.0):
     - Sentry (~> 7.25)
```

Comment:
@SiobhyB I noticed that several pods have been updated. However, I don't think this is expected when only bumping the Gutenberg Mobile reference. Usually, only the pods related to Gutenberg are the ones that are updated 🤔.

Reply:
Not sure if it's caused by this, but I'm getting the following error when building the app:
```
./WordPress-iOS/WordPress/Classes/Utility/Kanvas/KanvasCameraAnalyticsHandler.swift:5:20: error build: Type 'KanvasAnalyticsHandler' does not conform to protocol 'KanvasAnalyticsProvider'
```

Reply:
Hmmm, I'm not sure how those got updated along the way 🤔 this should be addressed in the most recent commits, though.

Reply:
Great, I've just checked the last changes and it looks good. Thanks!